code/game/objects/structures/spirit_board.dm

```diff
@@ -6,77 +6,99 @@
 	resistance_flags = FLAMMABLE
 	density = TRUE
 	anchored = FALSE
+	/// Whether no one has moved the planchette yet.
 	var/virgin = TRUE //applies especially to admins
-	var/next_use = 0
-	var/planchette = "A"
+	/// How long between planchette movements.
+	COOLDOWN_DECLARE(next_use)
+	/// Where the planchette is currently pointing.
+	var/planchette
+	/// Ckey of last mob to use the board.
 	var/lastuser = null
+	/// List of options ghosts (or people) can pick from.
+	var/list/ghosty_options = list(
+		"A","B","C","D","E","F","G","H","I","J","K","L","M","N","O","P","Q","R","S","T","U","V","W","X","Y","Z",
+		"1","2","3","4","5","6","7","8","9","0",
+		"Yes","No",
+	)
+	/// Admin only var, allows admins to bypass the check requiring multiple players
+	var/allow_oneman_use = FALSE
+
+/obj/structure/spirit_board/Initialize(mapload)
+	. = ..()
+	if(prob(1))
+		name = "luigi board"
+	planchette = ghosty_options[1]
 
 /obj/structure/spirit_board/examine()
-	desc = "[initial(desc)] The planchette is sitting at \"[planchette]\"."
 	. = ..()
+	. += span_notice("The planchette is currently at the letter \"[planchette]\".")
 
 /obj/structure/spirit_board/attack_hand(mob/user, list/modifiers)
 	. = ..()
 	if(.)
-		return
+		return .
 	spirit_board_pick_letter(user)
+	return TRUE
 
-
-//ATTACK GHOST IGNORING PARENT RETURN VALUE
 /obj/structure/spirit_board/attack_ghost(mob/dead/observer/user)
+	. = ..()
+	if(.)
+		return .
 	spirit_board_pick_letter(user)
-	return ..()
+	return TRUE
 
-/obj/structure/spirit_board/proc/spirit_board_pick_letter(mob/M)
-	if(!spirit_board_checks(M))
-		return FALSE
+/obj/structure/spirit_board/proc/spirit_board_pick_letter(mob/ghost)
+	if(!spirit_board_checks(ghost))
+		return
 
 	if(virgin)
 		virgin = FALSE
-		notify_ghosts("Someone has begun playing with a [src.name] in [get_area(src)]!", source = src, header = "Spirit board")
+		notify_ghosts("Someone has begun playing with \a [src] in [get_area(src)]!", source = src, header = "Spirit board")
 
-	planchette = tgui_input_list(M, "Choose the letter.", "Seance!", list("A","B","C","D","E","F","G","H","I","J","K","L","M","N","O","P","Q","R","S","T","U","V","W","X","Y","Z"))
+	planchette = tgui_input_list(ghost, "Choose the letter.", "Seance!", ghosty_options)
 	if(isnull(planchette))
 		return
```

Comment:
to be safe, let's not modify planchette directly through the `tgui_input_list`, but set a proc-local variable and then set planchette to that if it doesn't become null. feels more hygenic.